🚀 - Update Cypress to 12.15.0

### Which package(s) are relevant/related to the feature request?

Don't known / other

### Description

See [error logs](https://github.com/taiga-family/maskito/actions/runs/6286561265/job/17070207105?pr=527):

```
Still waiting to connect to Chrome, retrying in 1 second (attempt 58/62)
Still waiting to connect to Chrome, retrying in 1 second (attempt 59/62)
Still waiting to connect to Chrome, retrying in 1 second (attempt 60/62)
Still waiting to connect to Chrome, retrying in 1 second (attempt 61/62)
Still waiting to connect to Chrome, retrying in 1 second (attempt 62/62)
Cypress failed to make a connection to the Chrome DevTools Protocol after retrying for 50 seconds.

This usually indicates there was a problem opening the Chrome browser.

The CDP port requested was 34071.

Error: connect ECONNREFUSED 127.0.0.1:34071
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:11[87](https://github.com/taiga-family/maskito/actions/runs/6286561265/job/17070207105?pr=527#step:6:88):16)

Error: Process completed with exit code 1.
```

**Why?**
* https://github.com/cypress-io/cypress-documentation/issues/5479

--- a/package.json
+++ b/package.json
@@ -110,7 +110,7 @@
         "@types/node": "17.0.45",
         "babel-jest": "27.5.1",
         "cspell": "7.3.6",
-        "cypress": "10.11.0",
+        "cypress": "11.2.0",
         "cypress-real-events": "1.10.3",
         "eslint": "8.49.0",
         "eslint-config-prettier": "7.2.0",

--- a/package.json
+++ b/package.json
@@ -110,7 +110,7 @@
         "@types/node": "17.0.45",
         "babel-jest": "27.5.1",
         "cspell": "7.3.6",
-        "cypress": "11.2.0",
+        "cypress": "12.17.4",
         "cypress-real-events": "1.10.3",
         "eslint": "8.49.0",
         "eslint-config-prettier": "7.2.0",

